FeedReader.ConsoleSample/Program.cs

```diff
@@ -13,8 +13,7 @@ static void Main(string[] args)
                 string url = Console.ReadLine();
 
                 var urlsTask = FeedReader.GetFeedUrlsFromUrlAsync(url);
-                urlsTask.ConfigureAwait(false);
-                var urls = urlsTask.Result;
```

Comment:
If the Task faults with an exception, Task.Result will rethrow an AggregateException. It is better to call Task.GetAwaiter().GetResult() since it will rethrow an "unwrapped" exception.